tests/rules/method.js

```diff
@@ -830,6 +850,6 @@ eslintTester.run("method", rule, {
                     type: "FantasyCallee"
                 }
             ]
-        }
+        }        
```

Comment:
Nit, there seems to be some unnecessary whitespaces after the close braket here.